### DISCUSSION TITLE
Pixelmator Template outdated?

### DISCUSSION THREAD

**Discussion by marcmelliger** [elrumo/macOS_Big_Sur_icons_replacements#1297] [General]
I believe the Pixelmator template (teste wit Pixelmator 2.1.5) is outdated/suboptimal: https://macosicons.com/resources/macos-big-sur-app-icon

The issue is the large white shadow. This usually is not the case with the other icons. I'd send a new template but don't know the usual settings for the shadows in the icons.

**Comment by cake03** [comment]
The shadows are: 0 px, 10 px, 10 blur

**Reply by cake03** [reply]
and colour: `#000000` 30% Opacity

**Comment by DIntriglia** [comment]
@marcmelliger @C47F15H I have gone ahead and updated the Pixelmator Pro template to use an actual shadow effect as well to make it easier to users to update. Thanks for the settings @C47F15H 
[Template-Icon-App Updated.zip](https://github.com/elrumo/macOS_Big_Sur_icons_replacements/files/7445411/Template-Icon-App.Updated.zip)
